### DISCUSSION TITLE
ckb 0.111.0-rc12 (a15a467 2023-09-07)

### DISCUSSION THREAD

**Discussion by doitian** [nervosnetwork/ckb#4147] [Announcements]
This is an RC version to preview the changes in the next release.

### Compatibility Table

| Network | Consensus Version        | Minimal CKB Version |
|---------|------------------------- | ------------------- |
| dev     | ckb2023 since epoch 0    |                     |
| testnet | ckb2021 since epoch 3113 | v0.101.0            |
| mainnet | ckb2021 since epoch 5414 | v0.103.0            |

### Downloads

| OS                | Arch  | Runtime Dependencies | Package                 | Sign                |
| ----------------- | ----- | -------------------- | ----------------------- | ------------------- |
| macOS             | x64   | macOS 10.15 or above | [zip][macos-zip]        | [PGP][macos-pgp]    |
| macOS  (Portable) | x64   | macOS 10.15 or above | [zip][macos-p-zip]      | [PGP][macos-p-pgp]  |
| macOS             | ARM64 | macOS 10.15 or above | [zip][macosa-zip]       | [PGP][macosa-pgp]   |
| macOS  (Portable) | ARM64 | macOS 10.15 or above | [zip][macosa-p-zip]     | [PGP][macosa-p-pgp] |
| Linux             | x64   | glibc, libstdc++     | [tarball][linux-zip]    | [PGP][linux-pgp]    |
| Linux  (Portable) | x64   | glibc, libstdc++     | [tarball][linux-p-zip]  | [PGP][linux-p-pgp]  |
| CentOS            | x64   | glibc, libstdc++     | [tarball][centos-zip]   | [PGP][centos-pgp]   |
| CentOS (Portable) | x64   | glibc, libstdc++     | [tarball][centos-p-zip] | [PGP][centos-p-pgp] |
| Windows           | x64   | VC++ Redistributable | [zip][windows-zip]      | [PGP][windows-pgp]  |
| Linux             | ARM64 | glibc, libstdc++     | [tarball][arm-zip]      | [PGP][arm-pgp]      |

The `.asc` files are signatures. It is wise and more secure to check out for the files [integrity](https://github.com/nervosnetwork/ckb/blob/develop/docs/integrity-check.md).

Try the portable variants if the binaries complains about illegal instructions.

[macos-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-apple-darwin.zip
[macos-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-apple-darwin.zip.asc
[macos-p-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-apple-darwin-portable.zip
[macos-p-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-apple-darwin-portable.zip.asc
[macosa-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_aarch64-apple-darwin.zip
[macosa-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_aarch64-apple-darwin.zip.asc
[macosa-p-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_aarch64-apple-darwin-portable.zip
[macosa-p-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_aarch64-apple-darwin-portable.zip.asc
[linux-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-unknown-linux-gnu.tar.gz
[linux-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-unknown-linux-gnu.tar.gz.asc
[linux-p-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-unknown-linux-gnu-portable.tar.gz
[linux-p-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-unknown-linux-gnu-portable.tar.gz.asc
[centos-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-unknown-centos-gnu.tar.gz
[centos-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-unknown-centos-gnu.tar.gz.asc
[centos-p-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-unknown-centos-gnu-portable.tar.gz
[centos-p-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-unknown-centos-gnu-portable.tar.gz.asc
[windows-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-pc-windows-msvc.zip
[windows-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_x86_64-pc-windows-msvc.zip.asc
[arm-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_aarch64-unknown-linux-gnu.tar.gz
[arm-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.111.0-rc12/ckb_v0.111.0-rc12_aarch64-unknown-linux-gnu.tar.gz.asc

### Changes since v0.111.0-rc11

https://github.com/nervosnetwork/ckb/compare/v0.111.0-rc11...9fb20c1

<hr /><em>This discussion was created from the release <a href='https://github.com/nervosnetwork/ckb/releases/tag/v0.111.0-rc12'>ckb 0.111.0-rc12 (a15a467 2023-09-07)</a>.</em>

**Comment by joshyates1980** [comment]
I am attempting to RUN CKB full node with version 0.111.0-rc11, but have the following errors:


![1](https://github.com/nervosnetwork/ckb/assets/7255545/bb29deee-4347-4a73-9f5a-0fb232126ba1)

![2](https://github.com/nervosnetwork/ckb/assets/7255545/e66c8bea-a5f0-4999-8809-5697086844a4)

![3](https://github.com/nervosnetwork/ckb/assets/7255545/04357a57-9f55-4f27-913c-b263e782983d)


I received the same errors when I tried to RUN CKB init command with version **ckb_v0.110.1_x86_64-pc-windows-msvc**.

My goal is to run a full node on this old laptop 24/7.
Here are my laptop specs:

![4](https://github.com/nervosnetwork/ckb/assets/7255545/2525ffbb-f9f9-4982-a2d5-b501aeebc5fb)

**Comment by quake** [comment]
> I am attempting to RUN CKB full node with version 0.111.0-rc11, but have the following errors:

could you install the latest [Microsoft Visual C++ Redistributable for Visual Studio](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170) and try it again?

**Reply by joshyates1980** [reply]
Thank you, this corrected the issue and I can now RUN CKB full node.

**Comment by joshyates1980** [comment]
@quake , I have tried to sync full node for many days on my older computer.  After I restarted my computer, and attempted to run ckb --indexer, I received this error:

![image](https://github.com/nervosnetwork/ckb/assets/7255545/c30ba9b6-dc0c-43e6-b745-cbc3f10092cc)

How can I continue my full node, it's around 60%.

**Comment by quake** [comment]
@joshyates1980 it's a known issue of 0.110.1, it may take a long time to starting sync again after service restart, 2 ~ 15 mins, depends on your disk IO performance, could you wait a while to see if it's normal? And this bug has been fixed in 0.111, I think it will be released in this week.

**Reply by quake** [reply]
btw, `ckb-indexer` has been migrated into ckb already in recent recent releases, you may use `ckb run --indexer` to start the built-in indexer service.
